tool.go

```diff
@@ -132,9 +145,16 @@ func download(t *tool) error {
 
 func install(t *tool) error {
 	log("installing " + t.Repository)
+
+	err := ioutil.WriteFile(path.Join(t.gopath(), ".gitignore"), gitignore, 0664)
```

Comment:
We'll also need to write a `.gitignore` into `_tools/.gitignore` to ignore `_tools/bin`